modyn/tests/selector/internal/selector_strategies/downsampling_strategies/test_loss_downsampling_strategy.py

```diff
@@ -0,0 +1,43 @@
+import os
+import pathlib
+import tempfile
+
+from modyn.selector.internal.selector_strategies.downsampling_strategies import LossDownsamplingStrategy
+
+database_path = pathlib.Path(os.path.abspath(__file__)).parent / "test_storage.db"
+TMP_DIR = tempfile.mkdtemp()
+
+
+def test_init_loss():
+    # Test init works
+    strat = LossDownsamplingStrategy(
+        {
+            "ratio": 10,
+            "sample_then_batch": True,
+        },
+        1000,
+    )
+
+    assert strat.downsampling_ratio == 10
+    assert isinstance(strat.remote_downsampling_strategy_name, str)
+    assert strat.requires_remote_computation
+    assert strat.maximum_keys_in_memory == 1000
+
+
+def test_command_loss():
+    # Test init works
```

Comment:
(same question for the other tests as above, won't repeat everywhere)

Reply:
Same as above, fixed